build/debian-hyperkube-base/Makefile

```diff
@@ -19,11 +19,11 @@
 
 REGISTRY?=gcr.io/google-containers
 IMAGE?=debian-hyperkube-base
-TAG=0.3
+TAG=0.4
```

Comment:
I think we're going to sequence #53642 ahead of this PR, so maybe this should be 0.5 instead

Reply:
bumped to 0.5